node/pkg/reporter/node.go

```diff
@@ -0,0 +1,199 @@
+package reporter
+
+import (
+	"context"
+	"crypto/rand"
+	"errors"
+	"math/big"
+	"os"
+	"time"
+
+	"bisonai.com/orakl/node/pkg/db"
+	"bisonai.com/orakl/node/pkg/raft"
+	"bisonai.com/orakl/node/pkg/utils"
+
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"github.com/libp2p/go-libp2p/core/host"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	TOPIC_STRING    = "orakl-offchain-aggregation-reporter"
+	LEADER_TIMEOUT  = 5 * time.Second
```

Comment:
@nick-bisonai Can you please explain how is this `LEADER_TIMEOUT` different from the one in `node/pkg/aggregator/node.go`?

Reply:
it's same, leader does particular job in certain basis of time. and this defines the frequency

Reply:
Then, how about just defining it only at once place?

Reply:
Okay!